draft-ietf-add-split-horizon-authority.xml

```diff
@@ -311,8 +314,7 @@
             <li>"subdomains": An array containing the claimed subdomains, as
                 dot-separated names with the parent suffix already removed, in
                 canonical order.</li>
-            <li>"algorithm": The hash algorithm, identified by its IANA-registered
-                Mnemonic.</li>
+            <li>"algorithm": The hash algorithm, identified by the the ZONEMD Hash Algorithm.</li>
```

Comment:
In the ZONEMD registry, there is no column labeled "Hash Algorithm".  Instead, there are columns labeled "Value" (number), "Description" (string), and "Mnemonic" (string), any of which could be encoded in JSON.  I think the description here probably needs to be precise about which column we are using.

Reply:
Agreed, updated text as follows:
   *  "algorithm": The hash algorithm is represented as an unsigned integer from the ZONEMD Hash Algorithms registry ([RFC8976], Section 5.2).